session/session.php

```diff
@@ -190,7 +190,7 @@ function session_create_id($prefix) {}
 
 /**
  * Perform session data garbage collection
- * @return int number of deleted session data for success, FALSE for failure.
+ * @return int|false number of deleted session data for success, false for failure or 0 sessions cleared.
```

Comment:
I can't find `false for 0 sessions cleared` in documentation. Where do you found that?

Reply:
yes you are right. Could not clear sessions does not mean 0 sessions cleared